integration_tests/can_launch/.gitignore

```diff
@@ -0,0 +1,9 @@
+.DS_Store
```

Comment:
move integration_tests into the dev/ subdirectory, so that it's not visible at the top level. We're trying to adjust our directory structure so that the flutter package only has bin, dev, examples, and packages top-level directories.
cc @abarth